spec_2/scrabble.js

```diff
@@ -0,0 +1,57 @@
+const playedTiles = "AERETOXMYCNS_B"
```

Comment:
@EthanJStark, Dude thank you. Your comments were little bits of brilliance I got to add to my code and coding repertoire. This last one also inspired me to rewrite the whole thing but from a more object-oriented approach 👍